webapp/channels/src/components/advanced_text_editor/use_key_handler.tsx

```diff
@@ -158,6 +158,10 @@ const useKeyHandler = (
             }
         }
 
+        if ((Keyboard.isKeyPressed(e, KeyCodes.PAGE_UP) || Keyboard.isKeyPressed(e, KeyCodes.PAGE_DOWN)) && location === Locations.CENTER) {
+            (document.getElementById('post-list-virtualized') as HTMLElement).focus();
```

Comment:
A couple things here:
1. Instead of using the type assertion to say that the element isn't null, could you change it instead to `document.getElementById(...)?.focus()` instead? It's generally safer to do it that way since we want to make the `AdvancedTextEditor` more reusable in the future, and we want to avoid using `as` wherever possible since it can reduce that sort of error
2. It seems like we should probably return after this to ensure none of the other handlers will run then